### PR TITLE
ft(hc/accounts): Set periodic reporting

### PR DESCRIPTION
**Pull request by louicoder**
What does this PR do?
Gives the user the ability to be able to configure periodic reporting

Description of Task to be completed?
The system allows the user to select period of choice when they will be receiving reports about their checks. the periods include either monthly, weekly or daily 

How should this be manually tested?
run ./manage.py runserver.
run ./manage.py sendreports.
configure sendgrid environmental key using export SENDGRID_API_KEY=paste_api_key_here. Then
Create a new check and ping the url. After that, set reporting period and whether to receive reports or not from Accounts settings page. 
**Note** : 
To receive email reports, you must select the option "send me a summary of my checks" on the Account settings page.

Any background context you want to provide?
previously the default setting was that a user only gets notified monthly.

What are the relevant pivotal tracker stories?
[156577918](https://www.pivotaltracker.com/story/show/156577918)

### CODE REVIEW

**Comment by rogerokello** [comment]
Great work @louiCoder a lot of code you have written, In the file `hc/accounts/views.py`, consider creating a new line to separate your imports and library imports. For example, create a new line above `django.utils import timezone`  in the above file. This is just to make the code easier to read.

**Review by SoniaRMK — COMMENTED** [review]
@louiCoder great progress on this story. However, is it possible to write more tests asserting that the expected values are actually being passed and a test(s) to confirm that exceptions such as passing values other than those expected are properly handled?

**Comment by louicoder** [comment]
@rogerokello i have worked on that. Thanks for the heads up.

File: hc/accounts/views.py [hunk 1]
```diff
@@ -253,7 +256,7 @@ def unsubscribe_reports(request, username):
         return HttpResponseBadRequest()
 
     user = User.objects.get(username=username)
-    user.profile.reports_allowed = False
+    user.profile.reports_allowed = 30
```
Comment (Rwothoromo) [inline hc/accounts/views.py]: @louiCoder Great work done. However, this line of code is confusing `reports_allowed` = 30. Could it be more descriptive? Or is it that 30 reports are allowed?

**Review by emugaya — APPROVED** [review]

**Pull request closed without merging**